src/packages/core/components/input-upload-field/input-upload-field.element.ts

```diff
@@ -203,7 +203,9 @@ export class UmbInputUploadFieldElement extends FormControlMixin(UmbLitElement)
 				@change="${this.#onUpload}"
 				accept="${ifDefined(this.extensions?.join(', '))}"
 				?multiple="${this.multiple}">
-				<uui-button label="upload" @click="${this.#handleBrowse}">Upload file here</uui-button>
+				<uui-button label="upload" @click="${this.#handleBrowse}"
+					>${this.localize.term('media_clickToUpload')}</uui-button
+				>
```

Comment:
Please set this as the label rather than the default slot

```suggestion
				<uui-button 
				    label=${this.localize.term('media_clickToUpload')}
				    @click="${this.#handleBrowse}"></uui-button>
```